helm.sh/_posts/2018-09-07-new-gov-and-elections.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "New Governance And Elections"
+permalink: "/blog/new-gov-and-elections/"
+author: "Matt Farina"
+authorlink: "https://mattfarina.com"
+---
+
+When Helm moved from being under Kubernetes to a CNCF project there was a need to create a governance structure. While under Kubernetes we relied on Kubernetes governance. To handle this we setup a [provincial governance](https://github.com/helm/community/blob/aa0586011786dfbc3993e7edd959a841241c96e3/governance/provisional-governance.md) with a goal of creating a long term one. After a few months we are happy to announce that the new governance structure as been written and approved.
+
+The gist of the new governance is that it organizes those responsible into a couple groups (org and project maintainers), spells out their responsibilities, and provides for decision making processes. You can read all the details in the [governance doc (here)](https://github.com/helm/community/blob/master/governance/governance.md).
+
+## Selecting Org Maintainers
+
+The next step is to select the initial set of Org Maintainers. These individuals are responsible for elements such as the scope, vision, brand, code of conduct, owning security issues, finances, and other aspects of this nature. The project maintainers, as they have long been, are responsible for the technical decisions with regard to the codebases.
```

Comment:
what project maintainers are is a little confusing here.
Maybe something like:
`People who are responsible for the technical decisions with regard to the respective codebases under the Helm GitHub org (commonly referred to as core maintainers) are known collectively as Project Maintainers under Helm.`

Reply:
I revised this section to make the two types of maintainers more clearer :)